README.md

```diff
@@ -22,7 +22,9 @@ console.log(results);
 > Hint: Assuming there is a document with a name of "Bob", why does `results` not contain an author model on the second line?
 
 ```js
-// Your answer...
+// I'm not completely sure about this, but I think this is the correct answer because just using find will return all documents that have the name: "Bob"
+var results = AuthorModel.findOne({name: "Bob"});
```

Comment:
Because mongo/mongoose operations are async, we need to perform our data handling inside of a callback or promise:
```js
AuthorModel.findOne({name: "Bob"}, (err, results)=>{
    console.log(results);    
});
```